frontend/app/controllers/Event.scala

```diff
@@ -218,7 +226,14 @@ trait Event extends Controller with ActivityTracking {
       } yield {
         event.service.getOrder(oid).map { order =>
           trackConversionToThankyou(request, event, Some(order), Some(request.member))
-          Ok(views.html.event.thankyou(event, order)).discardingCookies(DiscardingCookie(eventCookie(event)))
+
+          request.session.get(isComplimentaryKey) foreach ( _ =>
```

Comment:
You are right, displaying the free ticket doesn't actually imply that one uses it. I am going to change the implementation and try to record the free event ticket usage based on the attendee data